WhaleSpotting/Controllers/UserController.cs

```diff
@@ -40,4 +42,18 @@ public IActionResult Create([FromBody] UserRequest newUser)
         var url = Url.Action("GetById", new { userId = user.Id });
         return Created(url, new UserResponse(user));
     }
+
+    [HttpGet("ListUsers")]
```

Comment:
We can actually just leave this empty as we won't have any other Get methods, apart from get by ID
```suggestion
    [HttpGet("")]
```